runtime/shared_common/shrinit.h

```diff
@@ -293,6 +299,7 @@ typedef struct J9SharedClassesOptions {
 #define HELPTEXT_ADJUST_MAXAOT_EQUALS OPTION_ADJUST_MAXAOT_EQUALS"<size>"
 #define HELPTEXT_ADJUST_MINJITDATA_EQUALS OPTION_ADJUST_MINJITDATA_EQUALS"<size>"
 #define HELPTEXT_ADJUST_MAXJITDATA_EQUALS OPTION_ADJUST_MAXJITDATA_EQUALS"<size>"
+#define HELPTEXT_LAYER_EQUALS OPTION_LAYER_EQUALS"<number>"
```

Comment:
There must be a space between strings to be concatenated: `OPTION_LAYER_EQUALS "<number>"`.
Newer compilers, (e.g. xlc 16) will complain.

Reply:
Fixed.